shared/actions/fs/index.js

```diff
@@ -283,32 +291,40 @@ function* pollSyncStatusUntilDone(): Saga.SagaGenerator<any, any> {
       let {syncingPaths, totalSyncingBytes, endEstimate}: RPCTypes.FSSyncStatus = yield Saga.call(
         RPCTypes.SimpleFSSimpleFSSyncStatusRpcPromise
       )
-      yield Saga.put(
-        FsGen.createJournalUpdate({
-          syncingPaths: (syncingPaths || []).map(Types.stringToPath),
-          totalSyncingBytes,
-          endEstimate,
-        })
-      )
-      console.log({
-        msg: 'songgao',
-        syncingPaths,
-        totalSyncingBytes,
-        endEstimate,
-      })
+      yield Saga.sequentially([
+        Saga.put(
+          FsGen.createJournalUpdate({
+            syncingPaths: (syncingPaths || []).map(Types.stringToPath),
+            totalSyncingBytes,
+            endEstimate,
+          })
+        ),
+
+        // Trigger folderListLoad and mimeTypeLoad for paths that user might be
+        // looking at. Note that we are not checking the syncingPaths here,
+        // because we are polling and things can slip through between
+        // SimpleFSSyncStatus calls. So instead just always re-load them on the
+        // same interval we are polling on journal status.
+        ...Array.from(refreshTagsFolderList).map(([_, path]) => Saga.put(FsGen.createFolderListLoad({path}))),
```

Comment:
Right but it's a heavy thing to poll. Surely we could poll just what is being uploaded...

Reply:
We'd need to figure out what's updated from the notification then. I guess we could be smarter about it by looking at paths that's being flushed, but then there's still the race where paths can get flushed between journal status calls, in which case we'd miss items. `folderListLoad` is nice since it gets a "snapshot" of the folder from Simple FS which is a good source of truth.

Reply:
Also looks like `Map.prototype.values()` returns an `Iterator`, and you can't call `map` on it ...

Reply:
The counts are so small that it doesn't matter, you can just leave it.